src/main/groovy/com/spidasoftware/schema/conversion/changeset/v12/CSAMaxWindTemperatureChangeset.groovy

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright (c) 2024 Bentley Systems, Incorporated. All rights reserved.
+ */
+package com.spidasoftware.schema.conversion.changeset.v12
+
+import com.spidasoftware.schema.conversion.changeset.ConversionException
+import com.spidasoftware.schema.conversion.changeset.client.AbstractClientDataChangeSet
+
+/**
+ * In version 24.1, the default temperature for CSA Maximum Wind load cases was -20 °C.
+ * CSA 60826 code updates the default temperature to 15 °C.
+ *
+ * When up-converting, we override the default temperature to be 15 °C.
+ * When down-converting, we remove the override.
```

Comment:
@kevankramb Should we remove the override when downconverting, or does this risk removing an override that the user intentionally created?